Use rummager results for image_urls

sihugh opened the pull request:
This eliminates up to three calls to content store for each page's navigation.

To do this we need to add image_url to the fields that rummager returns.

We can remove the alt text for images that are links because the link text is more relevant (and is preferred by browsers anyway).

We then need a default image in case the content doesn't have an image (not all do).

After rebasing on top of https://github.com/alphagov/government-frontend/pull/1043 and https://github.com/alphagov/government-frontend/pull/1032 it became clear that we can remove some things from the `news_and_communications` and `services` as they don't need to have content split into `promoted` and `tagged` content any more as they only display it in one form now.

I've therefore removed the `promoted` sections from these presenters which allows us to simplify some things and delete a bunch of code.

---

https://government-frontend-pr-1042.herokuapp.com/apply-apprenticeship

andrewgarner commented on app/presenters/supergroups/news_and_communications.rb:
```diff
@@ -26,22 +28,33 @@ def tagged_content
-          alt: document_image["alt_text"],
-          context: document_image["context"]
+    def format_document_data(documents, data_category: nil)
+      documents.each.with_index(1).map do |document, index|
```
I had to double check the Ruby API documentation to confirm that `1` is an offset. Could we extract this to provide a little more context?

sihugh replied:
👍

andrewgarner commented on app/presenters/supergroups/news_and_communications.rb:
```diff
@@ -2,6 +2,8 @@ module Supergroups
   class NewsAndCommunications < Supergroup
     attr_reader :content
 
+    PLACEHOLDER_IMAGE = "https://assets.publishing.service.gov.uk/government/assets/placeholder.jpg".freeze
```
Should we use Plek to make this environment specific?

```rb
PLACEHOLDER_IMAGE = URI::HTTPS.build(host: Plek.current.public_asset_host, path: '/government/assets/placeholder.jpg').freeze
```

sihugh commented:
I've updated this and rebased after a couple of merges.  It's worth sticking `?w=1` on the end of the URL when looking at the diff as Github's got a little confused about what is a change in a couple of places.
